test/integration/customUrlMoreOptions.spec.ts

```diff
@@ -0,0 +1,27 @@
+import { CustomUrlMoreOptionsClient } from "./generated/customUrlMoreOptions/src/customUrlMoreOptionsClient";
+import { assert } from "chai";
+import { CustomUrlMoreOptionsClientOptionalParams } from "./generated/customUrlMoreOptions/src/models";
+
+describe("Custom URL More Options", () => {
+  let client: CustomUrlMoreOptionsClient;
+
+  beforeEach(() => {
+    const clientOptions: CustomUrlMoreOptionsClientOptionalParams = {
+      endpoint: "http://localhost:3000",
+      dnsSuffix: ""
+    };
+    client = new CustomUrlMoreOptionsClient("test12", clientOptions);
+  });
+
+  describe("Paging", () => {
+    it("getPagesPartialUrl", async () => {
```

Comment:
It looks like this the describe/it text is copy-pasted from customUrlPaging. Maybe it could just be changed to
```suggestion
  describe("Paths", () => {
    it("getEmpty", async () => {
```

Reply:
🤦‍♂️ thanks! good catch